docs/pages/jobLog/viewing-job-logs.md

```diff
@@ -9,38 +9,38 @@ Project Explorer provides a view in the `IBM i Job Log` panel to display all of
 ### Project level
 The top level of this tree is the project name with the project description as its details.  This allows the job logs of multiple projects to be shown in this view.  Only the latest build is persisted in the project, but the other jobs are kept in memory until the workspace is restarted or the `Clear Previous Job Logs` action on the project is run.
 
-<img src="/assets/jl_action_clear.png" width="300">
+<img src="https://github.com/IBM/vscode-ibmi-projectexplorer/blob/main/docs/assets/jl_action_clear.png?raw=true" width="400">
```

Comment:
Why switch to URL and not use relative paths like in other locations of the doc?

Reply:
@SanjulaGanepola is it in order for the URL to work from the walkthrough as well?